KuringApp/KuringApp/Setting/OpenSourceView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  OpenSourceView.swift
+//  KuringApp
+//
+//  Created by 박성수 on 11/30/23.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+struct OpenSourceFeature: Reducer {
+    struct State: Equatable {
+        
+    }
+    
+    enum Action {
+        
+    }
+    
+    var body: some ReducerOf<Self> {
+        Reduce { state, action in
+            switch action {
+                
+            }
+        }
+    }
+}
+
+struct OpenSourceView: View {
+    let store: StoreOf<OpenSourceFeature>
+    
+    var body: some View {
+        WithViewStore(self.store, observe: { $0 }) { viewStore in
+            List(Opensource.items, id: \.self) { item in
```

Comment:
item 이 `Identifiable`를 준수하면 id 파라미터에 값을 할당하지 않아도 됩니다.
```suggestion
            List(Opensource.items) { item in
```